### DISCUSSION TITLE
Cannot install Firefox add-on - corrupt add-on?

### DISCUSSION THREAD

**Discussion by dataguy73301** [aclap-dev/video-downloadhelper#409] [Help and Q&A] [answered]
### Your question

Hello - when I try to install the add-on to Firefox I get an error message - "Installation aborted because the add-on appears to be corrupt."

I've reset Firefox (most up to date version 125.0.1) and created a new profile, but that did not work.  When I try to install the add-on from the .xpi file, I get an error message that I do not have permission to open the file, but I am the admin on this Windows 11 Pro desktop and have full control of the file.

I am not using a VPN.  I have paused my antivirus software (ESET).   And nothing has worked.  Does anyone have any suggestions?

Regards,
Mateo

### Addon & CoApp version and details

I don't have this information since the add-on is not installed...

**Comment by paulrouget** [comment]
Where do you get the addon/xpi from?

**Reply by dataguy73301** [reply, marked as answer]
UPDATE!!  My anti virus is ESET and found a solution on their forum.  Resetting all ESET to default solved the problem.  Thank you paulrouget for the help!!
